docs/career_assessment_how_to.md

```diff
@@ -0,0 +1,65 @@
+# Career assessment
+## How does it work?
+Each "Me" or "Not Me" response is matched with a trait like "Adaptable" or "Organized". Each career is associated with a few traits. When results are calculated, each career is given a score to reflect how closely it matches the traits chosen by the job seeker. The three careers with the highest score are displayed.
+
+Matching an image with a trait can happen three ways:
+1. Positive association: responding "Me" to an image is a "Me" response to the trait (Example: 'Working with customers' -> 'Customer service')
+2. Reverse association: responding "Not Me" to an image is a "Me" response to certain `reverse_trait`s (Example: 'Working with your hands' reverses to 'Technical')
+3. Compound association: Some traits are hard to encapsulate in a single image, so "Me" responses to multiple images can also become a "Me" response to a higher-level trait (Example: 'Teamwork' and 'Self-starter' are parts of the 'Leadership' trait)
+
+## How can I update the career information?
+The career info is managed via a google spreadsheet that is manually synced with the site's database. Updating existing career info or adding a new career doesn't involve any code updates, but adding a new trait or changing the assessment does.
+
+### The google spreadsheet
+The current production version of this site uses [this spreadsheet](https://docs.google.com/spreadsheets/d/1lMeZZ_vu-xIVKLgo7obKlF0X4iJ3oHkLKl8uIrPsSt8) (current owners CfA NOLA Fellows).
+
+*When editing the content DO NOT change the column headers or the content will not update properly.* The columns `description`, `	foundational_skills`, and `training` accept HTML formatting. It's recommended to double check your HTML with a tool like https://jsfiddle.net/.
+
+The numbers in the `traits` column correspond to the `id` column in the traits spreadsheet tab. To add or remove a trait from a career, add or remove the corrsponding `id` number from the `traits` column (make sure they are comma-separated). See below for adding an entirely new trait to the spreadsheet.
+
+#### Starting from scratch
+If you're setting up a new copy of this site, use this google spreadsheet template: https://drive.google.com/previewtemplate?id=1vgqsoXLJ9FSDZamGv-SzDX_xCV3wHY0Lh142nhuZDQI&mode=public. After filling in the info, go to 'File' > 'Publish to Web' and publish it as a web page. Make sure "Automatically republish when changes are made" is checked at the bottom. Save the link! You can also get the link again by opening the "Publish to Web" dialog again.
```

Comment:
Feels like there should be a new paragraph for a new idea: 

(new para) After filling in the info....